Mlem/Packages/MlemMiddleware/Sources/MlemMiddleware/InstanceConnection/PiefedConnection/PieFedConnection+Feature.swift

```diff
@@ -12,11 +12,11 @@ public extension PieFedConnection {
         try await Self.supports(feature, version: version)
     }
     
-    func supportsOrNil(_ feature: Feature) -> Bool? {
+    func supportsOrElse(_ feature: Feature, defaultValue: Bool) -> Bool {
```

Comment:
Having `defaultValue` as the argument name makes calls quite long. What about:

```suggestion
    func supportsOrElse(_ feature: Feature, default: Bool) -> Bool {
```

Or:

```suggestion
    func supports(_ feature: Feature, orElse default: Bool) -> Bool {
```

Reply:
I've refactored to `.supports(_ feature: Feature, defaultValue: Bool)`. Using `default` directly is awkward because it's a reserved keyword for switch statements; I think `defaultValue` makes the behavior clear enough without requiring the additional `orElse` label. I have also added a doc comment making the behavior explicit.